<?xml version="1.0"?>
<launch>
  <arg name="mav_name" default="firefly"/>
  <arg name="world_name" default="basic"/>
  <arg name="enable_logging" default="false"/>
  <arg name="image_def" default="vi_sensor/camera_depth/depth"/>

  <group ns="$(arg mav_name)">
      <node pkg="geometric_controller" name="firefly_PositionVelocityController" type="position_velocity_controller_R5_test.py" respawn="true"/>
      <node pkg="geometric_controller" name="polynomial_trajectory" type="polynomial_trajectory_generator.py" respawn="true" output="screen"/>
      <remap from="odometry" to="odometry_sensor1/odometry" />	
     
  </group>

</launch>
